lib/bank_accounts/savings_account.rb

```diff
@@ -0,0 +1,43 @@
+module BankAccounts
+  class SavingsAccount < Account
+
+    def initialize(id, initial_balance)
+      @id = id
+
+      # must have at least $10 balance
+      if initial_balance < 10
+        raise ArgumentError, "Savings account balance must be at least $10. Create a new savings account."
+      else
+        @balance = initial_balance
+      end
+    end
+
+
+    def withdraw(amount)
+      amount = amount.abs # real ATMs don't have a 'negative number' button
+      fee = 2
+      if (@balance - (amount + fee)) < 10
+        puts "Savings account balance must be at least $10. Transaction not processed."
```

Comment:
Ahh, the reqs only mentioned raising exceptions for the account initialization, but I should have extrapolated that to the other messages.